src/iotjs_module_process.cpp

```diff
@@ -130,7 +130,7 @@ static JResult WrapEval(const char* source) {
   int len2 = strlen(source);
   int len3 = strlen(wrapper[1]);
 
-  String code("", len1 + len2 + len3 + 1);
+  String code("", len1 + len2 + len3);
```

Comment:
There is already a fix related to this issue in a pending PR (#330). As far as I can remember, it does not even need an updated jerry. I'd suggest (or ask, actually) to get that landed first.